backend/compact-connect/lambdas/common-python/cc_common/config.py

```diff
@@ -115,5 +116,13 @@ def users_table_name(self):
     def fam_giv_index_name(self):
         return os.environ['FAM_GIV_INDEX_NAME']
 
+    @property
+    def expiration_date_resolution_timezone(self):
+        """
+        This is the timezone used to determine the expiration dates of licenses and privileges.
+        We anticipate that this may change in the future, so we have a configuration value for it.
+        """
+        return timezone('US/Eastern')
```

Comment:
Just to avoid Daylight Savings Time headaches, I'd suggest that we consolidate around UTC-4, rather than actual Eastern time. That puts our system at the latest American-ish time zone, but one that doesn't jump around twice a year.

Reply:
Now that we're using a basic UTC offset, we [don't actually need pytz anymore](https://docs.python.org/3.6/library/datetime.html#datetime.timezone.utcoffset).